backend/dal/Migrations/20210126015055_v01.06.03.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010603 : SeedMigration
```

Comment:
@goemen has a new DB Migration that will need to align with yours.  Whomever gets theirs merged first should use the `v01.06.02` or `v01.06.03`.